poetry.lock

```diff
@@ -133,7 +134,7 @@ optional = false
 python-versions = ">=3.5.0"
 
 [package.extras]
-unicode-backport = ["unicodedata2"]
+unicode_backport = ["unicodedata2"]
```

Comment:
There's a lot of unrelated changes, IIRC this happens if you're not on the latest poetry? (Or maybe if you have weird things in your poetry cache?)

Can you try clearing your cache and recreating this something like:

```bash
$ poetry cache list
$ poetry cache clear _default_cache --all
$ poetry cache clear pypi --all
$ poetry lock --no-update
```

I'm a bit confused at how this file was modified though. Did you do `poetry update cffi==1.15.1` and then `poetry lock`?

Reply:
Ah, yes. I ended up having to downgrade to poetry 1.2.0 to update the lock file, otherwise the docker image complained about the poetry lock file being too new. I don't know if we normally use poetry 1.3.0 to manipulate our lockfile though? And the solution is to update poetry in the docker image?

All I did here was run `poetry update cffi` on poetry 1.2.0.

Reply:
I think you want to use poetry `1.2.2` (one of the point releases changed something.)

Reply:
That sounds reasonable to me!